main.go

```diff
@@ -34,7 +34,7 @@ func main() {
 func start(
 	config *config.Config,
 	mapHandler mapping.HandlerInterface,
-	httpHandler http.Handler,
+	//httpHandler http.Handler,
```

Comment:
It's good practice to remove unused code.

Reply:
fixed in TSAPPS-176